apps/meteor/ee/app/license/server/license.ts

```diff
@@ -29,6 +34,26 @@ class LicenseClass {
 
 	private modules = new Set<string>();
 
+	private appsConfig: NonNullable<ILicense['apps']> = {
+		maxPrivateApps: 3,
+		maxMarketplaceApps: 5,
```

Comment:
These values will be hardcoded for now, right? I don't remember what was the decision on that

Reply:
Yes, these will be the default values for now and the license can override them